### PR TITLE
Update UG features

### PR DESCRIPTION
**Pull request by yashma-sonara**
Added setweak and summary feature in UG.

### CODE REVIEW

**Review by ronnnnnnnnn — APPROVED** [review]
lgtm

File: docs/UserGuide.md [hunk 1]
```diff
@@ -404,8 +401,41 @@ Furthermore, certain edits can cause the TeachStack to behave in unexpected ways
 
 ### 7.14 Setting weak threshold
```
Comment (JiaXinEu) [inline docs/UserGuide.md:402]: Do you think adding the command at the end would make this more consistent with the other features, ie.

### 7.14 Setting weak threshold : `setweak`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -404,8 +401,41 @@ Furthermore, certain edits can cause the TeachStack to behave in unexpected ways
...
+
+Default Weakness Threshold
+
+![](images/setWeakBefore.png)
+
+After command `setweak B`
+
+![](images/setWeakAfter.png)
+
 ### 7.15 Viewing summary statistics
```
Comment (JiaXinEu) [inline docs/UserGuide.md:424]: ### 7.15 Viewing summary statistics : `summary`

**Review by JiaXinEu — COMMENTED** [review]
Looks good, perhaps some changes on the sub-heading?

File: docs/UserGuide.md (hunk 1 above)
Comment (yashma-sonara) [reply docs/UserGuide.md:402]: Done in latest pr

**Pull request merged**